pkg/storage/endpoint/gc_safe_point.go

```diff
@@ -100,7 +100,9 @@ func (se *StorageEndpoint) LoadMinServiceGCSafePoint(now time.Time) (*ServiceSaf
 		}
 
 		if ssp.ExpiredAt < now.Unix() {
-			se.Remove(key)
+			if err := se.Remove(key); err != nil {
```

Comment:
Do we need to return the error here?

Reply:
fixed in https://github.com/tikv/pd/pull/8317/commits/c01b053b3bf93405d746c7db75db873345257ca3